src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/SupplementaryEvidenceMapper.java

```diff
@@ -41,6 +46,11 @@ public SupplementaryEvidence map(
         List<Map<String, Object>> existingEnvelopeReferences,
         Envelope envelope
     ) {
+        log.info(String.format("Mapping documents: container %s, zipFileName %s, caseRef %s",
```

Comment:
why are we creating string. can use logger {} ?

Reply:
Logger tests are failing otherwise